app/exceptions/personalization_exceptions.py

```diff
@@ -0,0 +1,6 @@
+
+class PersonalizationError(Exception):
+    """
+    Personalization Exception is raised when unable to personalize lineups or slates
+    """
+    pass
```

Comment:
Might be good to rename this file to personalization_error.py to stay consistent.